frontend/components/ActivityDialog.vue

```diff
@@ -539,8 +671,44 @@ function setSelectedDate(date: Date) {
                 />
             </div>
             <div
-                v-else-if="update"
-                class="mx-5 flex h-full flex-row justify-between gap-2 bg-background align-bottom font-nunito dark:bg-background-dark"
+                v-else-if="calendarClickedRef"
+                class="mx-5 flex h-full flex-row justify-between gap-1.5 bg-background align-bottom font-nunito dark:bg-background-dark"
+            >
+                <Button
+                    v-if="calendarActivity"
+                    type="button"
+                    :label="t('calendar.options.remove')"
+                    class="mt-auto h-9 w-40 rounded-xl border-2 border-cancel-border bg-input px-2 font-bold text-text hover:bg-cancel-bg dark:bg-input-dark dark:text-input dark:hover:bg-cancel-bg-dark"
```

Comment:
Wenn ich eine Aktivität in den Kalender Drag'n'Dropn will kann ich nicht gescheit runter scrollen. Die Seite springt jedes mal zurück